blog/content/post/2023-09-12-simplifying-firmware-update.md

```diff
@@ -9,7 +9,7 @@ abstract: "Updating your firmware is an important part of keeping your device
 cover: /covers/dasharo-sygnet.svg
 author: michal.kopec
 layout: post
-published: false
+published: true
 date: 2023-09-12
```

Comment:
@mkopec shouldn't we change the date? Or is it not related to the recent firmware releases?

Reply:
@artur-rs updated to today